.jsinspectrc

```diff
@@ -0,0 +1,8 @@
+{
+  "threshold":     60,
```

Comment:
This is not similar to codeclimate duplication weight. This number is the number of lines. A prudent value seems to be between 4 to 6.

Reply:
I'll make that change.